packages/truffle/test/scenarios/commandRunner.js

```diff
@@ -37,9 +37,24 @@ module.exports = {
       }
     });
   },
-  runInDevelopEnvironment: function (commands = [], config) {
-    const cmdLine = `${this.getExecString()} develop`;
-    const readyPrompt = "truffle(develop)>";
+  /**
+   * This is a function to test the output of a truffle develop/console command with arguments.
+   * @param {string[]} inputCommands - An array of input commands to enter when the prompt is ready.
+   * @param {TruffleConfig} config - Truffle config to be used for the test.
+   * @param {string} executableCommand - Truffle command to be tested (develop/console).
+   * @param {string} executableArgs - Arguments/options to be used with the executableCommand.
```

Comment:
Do you think it's worth stating these are space separated arguments?

Reply:
Yes, we can do that to make it more understandable.